InstanceNorm1D、InstanceNorm2D、InstanceNorm3D

Ayuan2021 opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->Others

### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->Docs

### Describe
<!-- Describe what this PR does -->修改了norm.py中InstanceNorm1D、InstanceNorm2D、InstanceNorm3D英文文档格式。函数说明处的中英文不一致问题。

Ayuan2021 committed:
modified:   python/paddle/nn/layer/norm.py

Ayuan2021 committed:
modified:   python/paddle/nn/layer/norm.py

Ayuan2021 committed:
modified:   python/paddle/nn/layer/norm.py

Ayuan2021 committed:
modified:   python/paddle/nn/layer/norm.py

Ayuan2021 committed:
modified:   python/paddle/nn/layer/norm.py

SigureMo commented:
需要按照提示过一下 CodeStyle Check～

```
2022-12-09 14:21:09 ************************************************************************************
2022-12-09 14:21:09 Your PR code style check failed.
2022-12-09 14:21:09 Please install pre-commit locally and set up git hook scripts:
2022-12-09 14:21:09 
2022-12-09 14:21:09     pip install pre-commit==2.17.0
2022-12-09 14:21:09     pre-commit install
2022-12-09 14:21:09 
2022-12-09 14:21:09 Then, run pre-commit to check codestyle issues in your PR:
2022-12-09 14:21:09 
2022-12-09 14:21:09     pre-commit run --files python/paddle/nn/layer/norm.py
2022-12-09 14:21:09 
2022-12-09 14:21:09 For more information, please refer to our codestyle check guide:
2022-12-09 14:21:09 https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/git_guides/codestyle_check_guide_cn.html
2022-12-09 14:21:09 ************************************************************************************
```

> test=docs_preview

这个需要在最新的 commit message 里加的，PR 描述信息里加是没有效果的

Ayuan2021 committed:
modified:   python/paddle/nn/layer/norm.py

Ligoml commented on python/paddle/nn/layer/norm.py:
```diff
@@ -62,7 +62,7 @@ def __init__(
         weight_attr=None,
         bias_attr=None,
-        data_format="NCHW",
+        data_format="NCL",
```
@ceci3 api负责人判断一下这样改是否ok？

ceci3 replied:
这里改了基类，会影响 InstanceNorm2D、3D的文档，不可以这样改哈

Ayuan2021 committed:
test=docs_preview

Ayuan2021 committed:
InstanceNorm2D中文档格式修改

Ayuan2021 committed:
test=docs_preview

Ligoml reviewed (dismissed):
good job！LGTM for docs
另外参考中文文档的pr，有两处细节可以补充修改：
paddle.nn.functional.instance_norm 英文文档增加对三个类函数的超链接
paddle.nn.functional.layer_norm 英文文档中的epsilon的参数介绍顺序需要调整

Ligoml commented:
paddle.nn.KLDivLoss 和 paddle.nn.functional.kl_div 的参数reduction需要增加optional标识

Ayuan2021 committed:
modified:   python/paddle/nn/functional/loss.py
	modified:   python/paddle/nn/functional/norm.py
	modified:   python/paddle/nn/layer/loss.py
	modified:   python/paddle/nn/layer/norm.py

Ligoml commented on python/paddle/nn/functional/norm.py:
```diff
@@ -327,19 +327,20 @@ def layer_norm(
     """
-    see more detail in paddle.nn.LayerNorm
+    nn.LayerNorm is recommended.
+    For more information, please refer to :ref:`cn_api_nn_LayerNorm` .
```
```suggestion
    For more information, please refer to :ref:`api_paddle_nn_LayerNorm` .
```

Ligoml commented on python/paddle/nn/functional/norm.py:
```diff
@@ -448,7 +449,7 @@ def instance_norm(
 ):
     """
-    See more detail in nn.layer.InstanceNorm2D.
+    It is recommended to use :ref:`cn_api_nn_InstanceNorm1D` , :ref:`cn_api_nn_InstanceNorm2D` , :ref:`cn_api_nn_InstanceNorm3D` to call this method internally.
```
```suggestion
    It is recommended to use :ref:`api_paddle_nn_InstanceNorm1D` , :ref:`api_paddle_nn_InstanceNorm2D` , :ref:`api_paddle_nn_InstanceNorm3D` to call this method internally.
```

Ligoml commented on python/paddle/nn/layer/norm.py:
```diff
@@ -127,25 +127,25 @@ class InstanceNorm1D(_InstanceNormBase):
             will create ParamAttr as weight_attr, the name of scale can be set in ParamAttr.
             If the Initializer of the weight_attr is not set, the parameter is initialized
-            one. If it is set to False, will not create weight_attr. Default: None.
+            one. If it is set to False, will not create weight_attr. Default: None. For more information, please refer to :ref:`cn_api_fluid_ParamAttr` .
```
```suggestion
            one. If it is set to False, will not create weight_attr. Default: None. For more information, please refer to :ref:`api_paddle_ParamAttr` .
```

Ligoml reviewed (commented):
英文文档和中文文档的标签略有不同，引用方式可以参考 [飞桨文档相互引用](https://github.com/PaddlePaddle/docs/wiki/%E9%A3%9E%E6%A1%A8%E6%96%87%E6%A1%A3%E7%9B%B8%E4%BA%92%E5%BC%95%E7%94%A8)

Ayuan2021 committed:
test=docs_preview

Ayuan2021 committed:
test=docs_preview

Ligoml reviewed (approved):
good job! LGTM for docs
